SU2_CFD/src/solvers/CIncEulerSolver.cpp

```diff
@@ -330,6 +331,15 @@ void CIncEulerSolver::SetNondimensionalization(CConfig *config, unsigned short i
       config->SetPressure_Thermodynamic(Pressure_Thermodynamic);
       break;
 
+    case FLUID_FLAMELET:
+
+      config->SetGas_Constant(UNIVERSAL_GAS_CONSTANT / (config->GetMolecular_Weight() / 1000.0));
+      Pressure_Thermodynamic = Density_FreeStream * Temperature_FreeStream * config->GetGas_Constant();
```

Comment:
From the Fluid-Mixture model , I have experienced the thermodynamic pressure influences the density at the inlet boundaries. So it would be better to provide the pressure_thermodynamic as a user value or a default value as 101325 [pa]. and based on this value compute the density.